### PR TITLE
Fix master compilation failure

### PR DESCRIPTION
**Pull request by MagoDelBlocco**
Fixes a few methods which were lost in merging #66, a few names and a few imports.

### CODE REVIEW

**Review by holounic — APPROVED** [review]
LGTM

File: src/test/java/com/google/graphgeckos/dashboard/DashboardApplicationTests.java [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package com.google.graphgeckos.dashboard;
 
-import org.junit.jupiter.api.Test;
+// import org.junit.jupiter.api.Test;
```
Comment (krasimir-google) [inline src/test/java/com/google/graphgeckos/dashboard/DashboardApplicationTests.java:3]: just delete this line if it is not required for the code to compile for now.

Comment (MagoDelBlocco) [reply src/test/java/com/google/graphgeckos/dashboard/DashboardApplicationTests.java]: Done!

**Review by krasimir-google — APPROVED** [review]
LGTM. Thank you!
Feel free to merge with master to address compilation failures.

**Pull request merged**